### ISSUE TITLE
Restrictions for (Candidate) Delegates

### ISSUE THREAD

**Issue by Laura-O** [thewca/wca-regulations#377]
As already discussed with the Board, we should add restrictions regarding the competition size and the number of Delegates delegating and the "type" of Delegates (candidate - no candidate).

First suggestion:
1. Competitions with more than 80 competitors should have at least one full Delegate.
2. Competitions with more than 200 competitors should have at least two Delegates.

**Comment by Claster** [comment]
Numbers seem to be reasonable, more or less (ideally they should depend on the previous experience of a Delegate, but that's too much for Regulations). Probably also:

3, If there is a moment when events are held in different rooms in parallel, each room should have a separate Delegate (candidate is ok) to watch the flow of the event (see also: 1c). Room is something that one Delegate can control (e.g., if one big hall has two scenes far away from each other, it is reasonable to have two Delegates controlling them).

**Comment by timreyn** [comment]
How many competitions have been held which do not follow these proposed regulations?  Are those competitions clustered in certain parts of the world?  It would be good to understand this before enacting these regulations, and make sure that those parts of the world have a way to move forward.

**Comment by Laura-O** [comment]
### Competitions with more than 80 competitors delegated by a Candidate Delegate (2016 only)

| Country | Freq |
| :-- | --: |
| USA | 9 |
| India | 5 |
| Brazil | 2 |
| Iran | 2 |
| Malaysia | 2 |
| Philippines | 2 |
| France | 1 |
| Poland | 1 |
| **Total** | **24** |

I used the WCA organizational spreadsheet to identify candidate delegates. This means that this statistic only includes competitions which were delegated by Candidate Delegates who are still candidates now. So if a Delegate was promoted last week, but delegated a competition with more than 80 competitors before that, this is not included here.

### Competitions with more than 200 competitors delegated by only one Delegate

| Country | Freq |
| :-- | --: |
| USA | 8 |
| India | 2 |
| Canada | 2 |
| China | 1 |
| Germany | 1 |
| Hungary | 1 |
| Poland | 1 |
| Russia | 1 |
| **Total**l | **17** |

This is rather pointless as it includes competitions like Worlds 2013, Euro 2012 and US Nationals 2012.

**Comment by AlbertoPdRF** [comment]
As this one is a Regulation for Delegates, maybe it makes more sense to put it on the Delegates Crash Course instead of the Regulations? A mention of it on the Organizer Guidelines also makes sense for me, just like the Proximity Police.

**Comment by Laura-O** [comment]
Yes, maybe this is also something the WCA Quality Assurance Committee should take care of (once it exists).

**Comment by Samuel-Baird** [comment]
What about situations where there are multiple candidate delegates? Here in Utah we currently have two candidate delegates and it wouldn't really make sense to limit us to 80 person competitions.

Another issue is it takes a year for a candidate delegate to become a delegate, I feel like it could really stunt community growth by limiting competition size. 

Maybe set a limit per candidate delegate and delegate? Perhaps 80 per candidate delegate and 200 per delegate, that allows for areas with multiple candidate delegates to still have a decent amount of competitors.

**Comment by Jambrose777** [comment]
This was discussed with Seniors and Leaders (and WQAC and WRC maybe?). The consensus was to not set a limit and if one is needed it can be imposed via probation.

**Comment by Ivan-Brigidano** [comment]
> This was discussed with Seniors and Leaders (and WQAC and WRC maybe?). The consensus was to not set a limit and if one is needed it can be imposed via probation.

Also, with WCRP this is no longer a Regulations topic
